fastparse/shared/src/test/scala/fastparse/WhiteSpaceMathTests.scala

```diff
@@ -7,19 +7,21 @@ import utest._
  * Same as MathTests, but demonstrating the use of whitespace
  */
 object WhiteSpaceMathTests extends TestSuite{
-  val White = WhitespaceApi.Wrapper{
+  val White = fastparse.WhitespaceApi.Wrapper{
```

Comment:
This is not needed here to compile, but as it is copied to the docs without the package context, one cannot just copy this piece of code (it's not clear where does this `WhitespaceApi` come from).